src-v2/Judge/NTS.Judge.Blazor/Ports/IParticipationBehind.cs

```diff
@@ -15,8 +15,8 @@ public interface IParticipationBehind : IObservableBehind
     // TODO: this should probably be removed and Participations can be returned from Start instead
     IEnumerable<Participation> Participations { get; }
     Participation? SelectedParticipation { get; set; }
-    Task RequestReinspection(bool requestFlag);
-    Task RequestRequiredInspection(bool requestFlag);
+    Task RequestRepresent(bool requestFlag);
```

Comment:
Maybe calling the action Represent isn't ideal because it has standalone meaning and isn't obvious we are referring to *presenting again*.

Reply:
But that is the FEI term, thus it's represented in our Domain Model and reflected elsewhere.